### PR TITLE
Updated routes for deployment on AWS

### PR DESCRIPTION
**Pull request by MichaelDurfey**
Trello: https://trello.com/c/iT1jOKc9/37-md-deploy-proxy-server-with-ebs
http://foodigov1.us-west-1.elasticbeanstalk.com/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 node_modules
+.DS_Store
+.gitignore
```
Comment (mattman240) [inline .gitignore:3]: are you gitignoring your gitignore?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <div id="pictures"></div>
+    <div id="sidebar"></div>
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+    <script src="http://foodigotitle-env.us-west-1.elasticbeanstalk.com/bundle.js"></script>
+    <script src="http://foodigoreviews.us-west-1.elasticbeanstalk.com/bundle.js"></script>
+    <script src="http://foodigopictures.us-west-1.elasticbeanstalk.com/bundle.js"></script>
+    <script src="http://foodigosidebar-env.us-east-2.elasticbeanstalk.com/bundle.js"></script>
+    <script>
+      let id = Math.floor((Math.random() * (200 - 101)) + 101);
```
Comment (mattman240) [inline public/index.html:30]: this is real cool and smart I might need to steal this idea

**Review by mattman240 — APPROVED** [review]
Looks good just a few comments

File: .gitignore (hunk 1 above)
Comment (MichaelDurfey) [reply .gitignore:3]: Yup, but I realize that doesn't do anything.. lol

File: public/index.html (hunk 2 above)
Comment (MichaelDurfey) [reply public/index.html:30]: Do it!

**Pull request merged**